pkg/controllers/scheduler/constants.go

```diff
@@ -45,6 +45,7 @@ const (
 	EventReasonInvalidFollowsObject      = "InvalidFollowsObject"
 	EventReasonWebhookConfigurationError = "WebhookConfigurationError"
 	EventReasonWebhookRegistered         = "WebhookRegistered"
+	EventReasonParseMigrationStrategy    = "ParseMigrationStrategy"
```

Comment:
We use `config` for the annotation and `strategy` here. Let's unify them to avoid confusing the users.

Reply:
Got.